_posts/2021-02-24-without-data-we-re-blind.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: post
+header_image: /img/posts/without-data-we-re-blind/header.jpg
+title: "Lifehacking 101: without data we're blind"
+author: Mauro Servienti
+synopsis: "Your day is a mess. By the end of the day, the predominant feeling is that you probably got nothing done. Before trying to address any problem, it's essential to understand which issues we're facing."
+tags:
+- Time management
+- To-Do lists
+- Context switching
+- Jorunaling
+---
+
+Your day is a mess. By the end of the day, the predominant feeling is that you probably got nothing done. Or maybe you got nothing done and wasted countless hours on low priority, not important tasks.
+
+If that's the case, you're not alone. That was me two years ago. Did I solve all my time management and planning problems? Far away from that. I did improve the situation a lot, though. Up to the point that I'm comfortable enough to say that I moved from a "fire-fighting" stage to a "what can we improve now?" phase. Big problems are a thing of the past. I'm now looking for opportunities rather than running after issues.
```

Comment:
```suggestion
If that's the case, you're not alone. That was me two years ago. Did I solve all my time management and planning problems? Far from it. I did improve the situation a lot, though. Up to the point that I'm comfortable enough to say that I moved from a "fire-fighting" phase to a "what can we improve now?" phase. Big problems are a thing of the past. I'm now looking for opportunities rather than running after issues.
```